src/groovy/org/pih/warehouse/inventory/StockMovementStatusCode.groovy

```diff
@@ -18,8 +18,10 @@ enum StockMovementStatusCode {
     CREATED(0),
     REQUESTING(1, PENDING),
     REQUESTED(2, PENDING),
+    WAITING_FOR_APPROVAL(2),
```

Comment:
Shouldn't this be `PENDING_APPROVAL`?
I saw Kacper has implemented these enums in #4288 so lets stick to the same name